README.md

```diff
@@ -100,17 +100,13 @@ Additional requirements for building on Windows:
 
 * [Cygwin](https://cygwin.com/install.html). It provides the "missing Unix pieces" required by build system such as rsync (and many others). Besides the default selected Unix tools such as `bash` and `rsync`, please also make sure to install `git` during Cygwin installation as well. You will need to clone this repository using `git` in Cygwin instead of the native Windows version of git, in order to ensure Unix line endings.
 
-To build the Electron clients, run:
+To build the Electron clients, run (it will also package an installer executable into `build/dist`):
```

Comment:
Should we move this out of the "Windows" block and into the "Electron" block?

Also, might as well wrap these in ```sh tags for consistency

Reply:
Make sense, updated.